cache-containers.html

```diff
@@ -123,57 +126,57 @@
                 </tr>
 
                 <tr>
-                    <td><span class="label label-success">AVAILABLE</span></td>
-                    <td><strong><a href="cache-container-caches.html">My cache container 1</a></strong></td>
-                    <td><span class="label label-default">113</span></td>
-                    <td><a href="clustering.html">JDG Cluster 1</a><br>
+                    <td style="padding: 10px"><span class="label label-success">AVAILABLE</span></td>
```

Comment:
I've tried that but by some reason it's not picked. When doing the actual implementation it can probably be easily done and tested.